test/e2e/apps/job.go

```diff
@@ -86,9 +88,20 @@ var _ = SIGDescribe("Job", func() {
 		Expect(err).NotTo(HaveOccurred())
 	})
 
+	It("should exceed active deadline", func() {
+		By("Creating a job")
+		var activeDeadlineSeconds int64 = 10
```

Comment:
Make it shorter, this will cause the entire test suite longer by 10 seconds. I'd propose 1-2 seconds max.